### DISCUSSION TITLE
column toggle in p-table not working

### DISCUSSION THREAD

**Discussion by Sujietha** [primefaces/community#1824] [PrimeNG] [answered]
attached stackblitz example. Column toggle is not working. pls check and help 
https://stackblitz.com/edit/mb6f17-d1e4bz?file=src%2Fapp%2Fdemo%2Ftable-column-toggle-demo.html,src%2Fapp%2Fdemo%2Ftable-column-toggle-demo.ts

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can only toggle dynamically generated columns. All of your columns are static, that's why you cannot toggle them. 
Here is a working example: https://stackblitz.com/edit/mb6f17-6kq86n?file=src%2Fapp%2Fdemo%2Ftable-column-toggle-demo.html,src%2Fapp%2Fdemo%2Ftable-column-toggle-demo.ts
